src/SourceGenerators/Uno.UI.SourceGenerators/Helpers/DependenciesInitializer.cs

```diff
@@ -33,7 +33,7 @@ internal static void Init()
 
 					var files = from file in Directory.EnumerateFiles(basePath, "*.dll")
 								let fileName = Path.GetFileName(file)
-								where fileName.StartsWith("Uno.")
+								where fileName.StartsWith("Uno.") || fileName == "Microsoft.CodeAnalysis.AnalyzerUtilities.dll"
```

Comment:
Why is this needed? Let's add a comment.

Reply:
I faced an error that this dll couldn't be found, so I loaded it here.